core/docker/default/etc/jvm.config

```diff
@@ -15,3 +15,8 @@
 # Improve AES performance for S3, etc. on ARM64 (JDK-8271567)
 -XX:+UnlockDiagnosticVMOptions
 -XX:+UseAESCTRIntrinsics
+# These are required for JDK 17 and have no affect on JDK[11,16)
```

Comment:
Why this is required? Do we plan to remove it? Do we need TODO here?

Reply:
We don't. It's required for Trino to work on JDK17